### DISCUSSION TITLE
Custom Sorting in Recent repositories view

### DISCUSSION THREAD

**Discussion by mobileFX** [gitextensions/gitextensions#11555] [Q&A] [answered]
### Feature description

Use Case:
Having to sync about 20 git repos (pull, merge, etc.)

Problem:
Recent repositories view sorts repos by last accessed causing confusion after a certain point of which repos are already processed and which repos are not processed.

Feature Proposition:
Provide custom sort (eg. sort by name) so that the list of repos remains the same.

### Environment

- GitExtensions version: latest
- GIT version: latest
- OS version: Windows 11
- .NET version: don't know

**Comment by pmiossec** [comment, marked as answer]
You should be able to do it with this setting:

![image](https://github.com/gitextensions/gitextensions/assets/460196/b29bb480-0776-40c0-8d6d-39a7c7d72a9b)

![image](https://github.com/gitextensions/gitextensions/assets/460196/1e9e76dd-9a3b-4218-9439-aa25e714cc9a)
